src/Microsoft.Sbom.Parsers.Spdx30SbomParser/Generator.cs

```diff
@@ -300,7 +292,7 @@ public GenerationResult GenerateJsonDocument(ExternalDocumentReferenceInfo exter
     /// <param name="relationship">Relationship info</param>
     /// <returns></returns>
     /// <exception cref="ArgumentNullException"></exception>
-    public GenerationResult GenerateJsonDocument(Relationship relationship)
+    public GenerationResult GenerateJsonDocument(Extensions.Entities.Relationship relationship)
```

Comment:
It seems weird that we have 2 different relationship classes being referenced in this file- Extensions.Entities.Relationship and Entities.Relationship. Why is that?

Reply:
One represents an SPDX 3.0 defined relationship, while the other represents an SBOM contract defined relationship. Both represent the same information but in different formats.

I considered naming one of them SPDXRelationship but that doesn’t follow the naming standards for SPDX 3.0 entities and makes things a bit complicated for parsing and validation since we rely on the class name being “Relationship” during json serialization.

Reply:
I’m open to any suggestions that might make this less confusing

Reply:
I renamed the assemblies so that hopefully makes the readability better.